packages/ember-data/lib/serializers/embedded_records_mixin.js

```diff
@@ -123,8 +123,10 @@ var EmbeddedRecordsMixin = Ember.Mixin.create({
     return extractEmbeddedRecords(this, this.store, type, normalizedHash);
   },
 
-  keyForRelationship: function(key, type) {
-    if (this.hasDeserializeRecordsOption(key)) {
+  keyForRelationship: function(key, type, method) {
+    if (method === undefined) { method = 'deserialize'; }
```

Comment:
I think the convention is to put the conditional body on its own line so its easier to inset a breakpoint if needed,